.github/workflows/tests.yml

```diff
@@ -41,20 +43,25 @@ jobs:
           - windows
         python:
           - '3.7'
+          - '3.8'
+          - '3.9'
+          - '3.10'
           - '3.11'
+          - '3.12-dev'
+          - 'pypy-3.9'
         meson:
           -
         include:
-          - os: ubuntu
-            python: 'pypy-3.9'
-          - os: ubuntu
-            python: '3.8'
-          - os: ubuntu
-            python: '3.9'
-          - os: ubuntu
-            python: '3.10'
-          - os: ubuntu
-            python: '3.12-dev'
+        #   - os: ubuntu
+        #     python: 'pypy-3.9'
+        #   - os: ubuntu
+        #     python: '3.8'
+        #   - os: ubuntu
+        #     python: '3.9'
+        #   - os: ubuntu
+        #     python: '3.10'
+        #   - os: ubuntu
+        #     python: '3.12-dev'
```

Comment:
```suggestion
        #     python: '3.12'
```

Reply:
I don't think that modifying commented out configuration has any effect.

Reply:
The reason that `allow-prereleases` was created was because the upgrade to production releases is done automatically when they ship instead of having to do that manually.  There is no difference today but there is a difference when Python 3.12 ships.

Reply:
That line is commented out. It could very well specify `3.99`.

Reply:
You are correct.  I did a find and replace.  Sorry for the oversight.